templates/auto-value/Targets.cshtml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    @foreach (var art in @Model.MavenArtifacts) {
+    <AndroidJavaLibrary Include="$(MSBuildThisFileDirectory)..\..\jar\@(art.MavenArtifactId)-@(art.MavenArtifactVersion).jar" />
```

Comment:
Anywhere you use `@(AndroidJavaLibrary)` make sure you check `$(AndroidApplication)` like we do here:

https://github.com/xamarin/AndroidX/blob/f97553ff428f9b6ea754f173567d220048245a16/source/AndroidXTargets.cshtml#L51

Otherwise developers' class libraries will "redistribute" this file.

Reply:
yes. Working on that and not all artifacts in template sets are jars!!!! (BuildActions)

Reply:
here:

```
https://github.com/xamarin/AndroidX/blob/f97553ff428f9b6ea754f173567d220048245a16/source/AndroidXTargets.cshtml#L47-L53
```
`*.aar` wil not be redistributed? Or do libraries remove `*.aar`s?

Reply:
@jpobst @jonathanpeppers 

@jpobst You added `kotlin` and `kotlinx` tamplate sets and they have `@(AndroidJavaLibrary)`. Should I (can I) fix that too?

Reply:
`@(AndroidAarLibrary)` is fine because it doesn't do anything in a class library. I would fix `@(AndroidJavaLibrary)` to check for `$(AndroidApplication)` if you find it in other places.

Reply:
> @jpobst You added kotlin and kotlinx tamplate sets and they have @(AndroidJavaLibrary). Should I (can I) fix that too?

Yes, looks like those use `<InputJar>` and package a loose `.jar`, so they should be updated.